### PR TITLE
Minor fixes for actions and development internal doc errors

### PR DESCRIPTION
**Pull request by TyDunn**
**Proposed changes**:
- Add `rasa run actions` to the `Actions` docs
- Update how to install documentation dependencies

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

**Comment by erohmensing** [comment]
@TyDunn a couple things -- first, please sign the CLA. Second, looks like you ran into doc build errors (which are actually probably just warnings but we treat them like errors). You can check these by making the livedocs locally `make livedocs` and then addressing any warnings in the console

File: docs/core/actions.rst [hunk 1]
```diff
@@ -84,15 +84,23 @@ The only thing your action server needs to install is ``rasa-sdk``:
...
 
 .. code-block:: bash
 
-    python -m rasa_sdk --actions actions
+    rasa run actions
 
 .. _custom_action_example:
 
+Otherwise, if your actions are defined in a file
+called ``actions.py`` and you do not have ``rasa`` installed, run this command:
```
Comment (tabergma) [inline docs/core/actions.rst]: Actually, if you run `rasa run actions` your actions should also be defined in `actions.py`. If the file as a different name, such as `custom_actions.py`, you need to specify it via `rasa run actions --actions custom_actions`.

Comment (TyDunn) [reply docs/core/actions.rst]: Okay, so do you think I should add that to the documentation? I just tried to follow the original and add that you can also use `rasa run actions` (which is completely missing right now); however, we need to keep the part about how to run your action server without ``rasa`` being installed, which was the main point of this section (not that your action need to be defined in ``actions.py``)

Comment (tabergma) [reply docs/core/actions.rst]: As it is not stated anywhere else, I guess it makes sense to mention that your actions file should be called `actions.py`. But would be great if you mention it at the beginning as it applies to both commands.

**Pull request merged**